Assets/IATK/Scripts/Model/CSVDataSource.cs

```diff
@@ -31,7 +31,7 @@ public class CSVDataSource : DataSource
             get { return textualDimensionsListReverse; }
             set { textualDimensionsListReverse = value; }
         }
-
+        

```

Comment:
I am sorry for that, that dropped in accidently from merging back into the iatk repo, since i have to run with the full project. VS2019 seems to like to much of win-lin linebrake, especially the autoformatter. I will go over this within the next days.